extensions/wikia/VideosModule/Modules/VideosModuleBase.class.php

```diff
@@ -122,7 +122,6 @@ public function getVideos() {
 	protected function getCacheKey() {
 		$cacheKey = wfSharedMemcKey(
```

Comment:
I believe you're referring to [this](https://github.com/Wikia/app/pull/5871#issuecomment-68807336) comment? My take away from that is the motivation is to make sure the proper cachekey prefix is used to differentiate environments. Using wfSharedMemcKey still accomplishes that (it has the same logic as wfMemcKey) so I'm of the opinion this implementation is probably fine.

The problem is too that if we called wfMemcKey in the base class, the staff picks and categories module would have to strip off the dbname from the key since we don't want those added there. Keeping it general and adding specificity for the one module that needs it (VideosModuleRelated.class.php) seems to make more sense to me in this case.